### PR TITLE
chore: bug report issue template

### PR DESCRIPTION
**Pull request by hifabienne**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:1]: Also interesting is the target environment:
- ZITADEL Cloud
- Self Hosted ZITADEL Version

File: .github/ISSUE_TEMPLATE/improvement
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/improvement:1]: add .yaml file ending

File: .github/ISSUE_TEMPLATE/improvement [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: 🛠️ Improvement
+description: 
+title: ""
+labels: ["type: improvement"]
```
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/improvement:4]:
```suggestion
labels: ["improvement"]
```

File: .github/ISSUE_TEMPLATE/improvement
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/improvement:1]: How does it differ from Proposal / Feature Request?

**Review by eliobischof — COMMENTED** [review]
Let's require a user story in all the templates? As <role>, I want...

File: .github/ISSUE_TEMPLATE/improvement
Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/improvement:1]: For me a feature request is something new. and an improvement is something we already have but should change in a way but is not a bug.

example: 
- Feature Request: Adding groups possibilty
- Improvement: The redirect urls of apps should be added the same way as servers on ldaps. 

Not sure if we really need both. I just had the feeling that customers sometimes don't know where to add.

Comment (eliobischof) [reply .github/ISSUE_TEMPLATE/improvement:1]: Having more choices makes the decision harder, doesn't it? 😄

What about adding " / Improvements" to the other template?

Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/improvement:1]: A feature is something new and a improvement something existing that should be changed

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:1]: The terraform provider, you can't setup instances with the provider in our cloud.

File: .github/ISSUE_TEMPLATE/improvement
Comment (eliobischof) [reply .github/ISSUE_TEMPLATE/improvement:1]: @stebenz do you think we should have a "Proposal / Feature request" issue template as well as an "Improvement" template?

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:1]: Let's add a required "impact" textarea field?
Something like
*"Describe your problems caused by this bug"*

File: .github/ISSUE_TEMPLATE/proposal.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - label:
+          I could not find a solution in the existing issues, docs, nor discussions
+        required: true
+      - label:
+          I have joined the [ZITADEL chat](https://zitadel.com/chat)
+  - type: textarea
+    id: problem
+    attributes:
+      label: Describe your problem
+      description: Is your proposal / feature request related to a problem. Please describe it.
```
Comment (eliobischof) [inline .github/ISSUE_TEMPLATE/proposal.yaml:24]: We don't care if it doesn't solve a real problem, right?
```suggestion
      description: Please describe your problem this proposal / feature is supposed to solve.
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  id: expected
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen.
+    placeholder: As a [type of user], I want [some goal] so that [some reason].
+- type: input
+  id: version
+  attributes:
+    label: Version
+    description: Which version of the Terrafor Provider are you using.
```
[the remark this answers is not in the document]
Comment (hifabienne) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: I changed the description field to impact, and changed the description to your suggestion.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (eliobischof) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:1]: I see you replaced the description field :+1:

**Review by eliobischof — APPROVED** [review]

**Pull request merged**